docs/user-guide/DREAM-SANS-methodology.ipynb

```diff
@@ -0,0 +1,56 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {},
+   "source": [
+    "# DREAM-SANS methodology\n",
+    "\n",
+    "The general data reduction scheme presented in \"SANS Polarization Analysis Methodology\" will be adapted for the SANS option at the DREAM instrument at the ESS. Following considerations have to be changed with respect to the general methodology:\n",
```

Comment:
> Following considerations have to be changed with respect to the general methodology:

I'm not sure I understand what this means.
I understood it as "some things in the SANS workflow we currently have will have to be changed so we can accommodate the DREAM instrument. And we are giving the list of those things below."
Is that correct?

Reply:
Yes, exactly